_studio/shared/include/mfx_utils.h

```diff
@@ -125,6 +127,18 @@ template<class T> inline T AlignValue(T value, mfxU32 alignment)
     return static_cast<T>((value + alignment - 1) & ~(alignment - 1));
 }
 
+namespace mfx
+{
+// TODO: switch to std::clamp when C++17 support will be enabled
+
+// Clip value v to range [lo, hi]
+template<class T>
+const T clamp(const T v, const T lo, const T hi)
```

Comment:
What for const for passes by value parameters?

Reply:
actually, ```const``` can be dropped, yes